index.html

```diff
@@ -50,5 +51,237 @@ <h4>LET'S CONNECT</h4>
       </ul>
     </section>
     <!--end of headline design-->
+
+    <!--card containers-->
+    <!--card 1-->
+    <section class="card-container">
+      <div class="card-image">
+        <img
+          src="background-svg\Snapshoot Portfolio.svg"
+          alt="image"/>
+      </div>
+      <div>
+        <h2 class="project-title">Tonic</h2>
+      </div>
+      <div class="texts">
+        <h3 class="bold-text">CANOPY</h3>
+        <div class="dot"></div>
+        <p class="text-1">Back End Dev</p>
+        <div class="dot"></div>
+        <p class="text-2">2015</p>
+      </div>
+<div>
+  <p class="primary-text">
+    A daily selection of privately personalized reads; no accounts or
+    sign-ups required.
+  </p>
+</div>
+
+<div class="tags-div">
+<ul class="tags">
+  <li class="html">HTML</li>
+</ul>
+<ul class="tags">
+  <li class="css">CSS</li>
+</ul>
+<ul class="tags">
+  <li class="js">JavaScript</li>
+</ul>
```

Comment:
- [ ] It could be better to use one `ul` tag that contains three `li` tags for the best practices of HTML & CSS 👇🏽 
- [ ] Please 🙏🏽 correct the styles, color, margins, padding and position to be similar to the Figma template.

```html
<ul>
  <li>HTML</li>
  <li>CSS</li>
  <li>JavaScriptL</li>
</ul>
```

| Figma template | Your design |
|---|---|
| ![image](https://github.com/JoyTaribagshaw/My-Portfolio/assets/58642949/ef1cc6e4-c45b-45e2-9e73-dd74b434c409) | ![image](https://github.com/JoyTaribagshaw/My-Portfolio/assets/58642949/f6cfb63f-0d86-4a53-8324-7a32176f44cf) |